torch/_torch_docs.py

```diff
@@ -7662,7 +7662,24 @@ def merge_dicts(*dicts):
 
 This function returns a namedtuple ``(U, S, V)`` which is the singular value
 decomposition of a input real matrix or batches of real matrices :attr:`input` such that
-:math:`input = U \times diag(S) \times V^T`.
+:math:`input = U \times diag(S) \times V^T`, where :math:`V^T` is the transpose
+of ``V``.
+
+The original tensor can be reconstructed by::
+
+    U @ diag(S) @ V.T
+
+
+.. note:: It is worth noting that that the code above works unmodified even
+          for complex numbers, i.e. the returned matrix ``V`` is already
+          conjugated.  This behavior is probably unexpected from the
+          mathematical point of view, but it is not possible to change it
+          without breaking existing code.  New code is encouraged to use
+          ``torch.linalg.svd`` instead, which returns :math:`V^H` instead.
+
+
+The dtype of ``U`` and ``V`` is the same as the ``input`` matrix. The dtype of
+``S`` is always real numbers, even if ``input`` is complex.
```

Comment:
For this bit I suggest: 

"The dtypes of \`\`U\`\` and \`\`V\`\` are the same as :attr:\`input\`'s. \`\`S\`\` will always be real-valued, even if :attr:\`input\` is complex."

Reply:
f16f6e75c5